internal/provider/provider.go

```diff
@@ -210,12 +224,20 @@ func (p *IncusProvider) Configure(ctx context.Context, req provider.ConfigureReq
 	// }
 	envName := os.Getenv("INCUS_REMOTE")
 	if envName != "" {
+		var env_public bool
+		if os.Getenv("INCUS_PUBLIC") == "true" || os.Getenv("INCUS_PUBLIC") == "True" {
```

Comment:
If this is just a string, then I would call the "lowercase" method for os.Getenv("INCUS_PUBLIC") so that the value is always `"true"`. Better would be if it could be parsed to a boolean.